.github/workflows/test.yml

```diff
@@ -0,0 +1,57 @@
+name: Test
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  changes:
+    runs-on: ubuntu-latest
+    # Required permissions
+    permissions:
+      pull-requests: read
+    outputs:
+      # Expose matched filters as job 'packages' output variable
+      packages: ${{ steps.filter.outputs.changes }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: dorny/paths-filter@v3
+        id: filter
+        with:
+          filters: |
+            exoflex: packages/exoflex/**
+            feetch: packages/feetch/**
+            flexship: packages/flexship/**
+            miflex: packages/miflex/**
+  test:
+    needs: changes
+    strategy:
+      matrix:
+        # Parse JSON array containing names of all filters matching any of changed files
+        # e.g. ['package1', 'package2'] if both package folders contains changes
+        package: ${{ fromJSON(needs.changes.outputs.packages) }}
```

Comment:
woops, seems like this doesn't get parsed correctly
<img width="876" alt="image" src="https://github.com/kodefox/infra/assets/24470609/f4b90be0-f0c3-4cb5-8484-eecf9301073e">